setup.cfg

```diff
@@ -30,7 +30,14 @@ zip_safe = False
 include_package_data = True
 packages = find:
 install_requires =
-    django >= 2.2
+    django>=2.2.0,<4.0
+    django-solo
+    djangorestframework~=3.12.0
+    djangorestframework_camel_case>=1.2.0
+    django-rest-framework-condition
+    gemma-zds-client>=0.14.0
+    requests
```

Comment:
this is a transitive dependency of gemma-zds-client already, so I think we can leave it out of this list.